### PR TITLE
glass_brain: fix colorbar

### LINKED ISSUE #379: Show colorbar with plot_glass_brain?
Is this possible?  I only see true/false, and I only see this option for plot_stat_map, not plot_glass_brain for instance.

Yet, I can manipulate all the colormap properties fine:

f, axarr = plt.subplots(20)  
f.set_size_inches(10,40)  #(width, length)[inches]

nilearn.plotting.plot_glass_brain(os.path.join(file_path, file_name), 
                                  axes=axarr[0], cmap=cmx.rainbow, vmin=-8, vmax=8, threshold = 4)

### PR DESCRIPTION
**Pull request by banilo**
tries to fix #379 

It's a quick fix for now that handles 0 to positive valued colorbars. Do we want more than that?

### CODE REVIEW

File: nilearn/plotting/displays.py [hunk 1]
```diff
@@ -501,10 +501,13 @@ def _colorbar_show(self, im, threshold):
         nb_ticks = 5 if im.norm.vmin != im.norm.vmax else 1
         ticks = np.linspace(im.norm.vmin, im.norm.vmax, nb_ticks)
         bounds = np.linspace(im.norm.vmin, im.norm.vmax, our_cmap.N)
-
+
```
Comment (AlexandreAbraham) [inline nilearn/plotting/displays.py]: :-1:

File: nilearn/plotting/img_plotting.py [hunk 2]
```diff
@@ -726,13 +728,25 @@ def plot_glass_brain(stat_map_img,
     def display_factory(display_mode):
         return functools.partial(get_projector(display_mode), alpha=alpha)
 
+    # set vmin/vmax by hand
+    stat_map_img = _utils.check_niimg(stat_map_img, ensure_3d=True)
+    stat_map_data = stat_map_img.get_data()
+    # Avoid dealing with masked_array:
+    if hasattr(stat_map_data, '_mask'):
+        stat_map_data = np.asarray(stat_map_data[
+                np.logical_not(stat_map_data._mask)])
```
Comment (AlexandreAbraham) [inline nilearn/plotting/img_plotting.py]: If it's just for computing the max, you don't need to unmask. If you really want to get the non-masked values, use `stat_map_data.compressed()`

**Comment by banilo** [comment]
Cleanups done!

**Comment by lesteve** [comment]
@banilo you have some tests failures that seem genuine.

**Comment by lesteve** [comment]
Add a test for plot_glass_brain with colorbar=True and squash your commits and this will be good to go.

**Comment by banilo** [comment]
> Add a test for plot_glass_brain with colorbar=True and squash your commits and this will be good to go.

done.

**Comment by lesteve** [comment]
Thanks!

**Pull request merged**